pkg/asset/machines/master.go

```diff
@@ -402,6 +402,14 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		}
 		machineConfigs = append(machineConfigs, ignFIPS)
 	}
+	if len(ic.Workload) > 0 {
+		w := ic.Workload[0]
+		ignWorkload, err := machineconfig.ForWorkloadPartitioning(string(w.Name), w.CPUIds, "master")
+		if err != nil {
+			return errors.Wrap(err, "failed to create ignition for workload partitioning for master machines")
```

Comment:
Add context for which partition had problems.
```suggestion
			return errors.Wrapf(err, "failed to create ignition for workload partitioning %q for master machines", w.Name)
```